Makefile

```diff
@@ -433,6 +433,8 @@ generate_grafana_scripts:
 bazel_ci_prepare:
 	bazel $(BAZEL_GLOBAL_CONFIG) run $(BAZEL_CMD_CONFIG) //:gazelle
 	bazel $(BAZEL_GLOBAL_CONFIG) run $(BAZEL_CMD_CONFIG) //:gazelle -- update-repos -from_file=go.mod -to_macro DEPS.bzl%go_deps  -build_file_proto_mode=disable
+	bazel $(BAZEL_GLOBAL_CONFIG) run $(BAZEL_CMD_CONFIG)  //cmd/mirror:mirror -- --mirror> tmp.txt
+	mv tmp.txt DEPS.bzl
```

Comment:
Maybe something like `$tmp=$(mktemp bzltmp.XXXXX)` would be better and not leave temp files around? Otherwise we might to add `tmp.txt` to the `.gitignore`.

Reply:
Done

Reply:
Looks like no all `> tmp.txt` lines were replaced?

Reply:
I think what you mentioned is "bazel_ci_prepare". It was actually missed, but it's not a big problem because it only runs in CI.